classes/author.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'book'
```

Comment:
- [ ] I think it would be better to add the `require_relative 'item'` to adhere to project requirements and the UML diagram
![Screenshot 2023-06-13 175205](https://github.com/AbdusSattar-70/Ruby-Group-Capstone/assets/118761303/1085d3e8-424b-4c9a-a011-9ea91868a9fb)

Reply:
The required change is implemented, thanks